spec/metric_fu/metrics/reek/generator_spec.rb

```diff
@@ -109,11 +102,6 @@
 
       it "should NOT insert nil smells into the array when there's a newline in the method call" do
         expect(@matches.last[:code_smells]).to eq(@matches.last[:code_smells].compact)
-        expect(@matches.last).to eq(file_path: "app/controllers/newline_controller.rb",
```

Comment:
Why was this test removed? I would think it's better to keep it to ensure new lines don't break with future changes.